### PR TITLE
Test/chalkboard tests

### PR DESCRIPTION
**Pull request by lilianaweimer**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- Chalkboard had no tests

## What is the new behavior?

- There are now a few passing tests for Chalkboard 

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

## Other information

The `rtlRender` might be better to put in a helper file and I couldn't figure out how to get the store to update properly; it won't change the score when the button is "clicked" (this is likely because I mocked the store, but I don't know how to get the mock to change).

### CODE REVIEW

File: src/Chalkboard/Chalkboard.test.js [hunk 1]
```diff
@@ -1,28 +1,132 @@
...
+import { render as rtlRender, fireEvent } from '@testing-library/react';
 import { BrowserRouter } from 'react-router-dom';
 import '@testing-library/jest-dom/extend-expect';
 import Chalkboard from './Chalkboard';
 import { Provider } from 'react-redux';
 import { createStore } from 'redux';
 
-const store = createStore(() => ({playerName: 'test', questions: 'test'}))
 
+const render = (
```
Comment (Billwilke42) [inline src/Chalkboard/Chalkboard.test.js:11]: Nice work on this!

File: src/Chalkboard/Chalkboard.test.js [hunk 2]
```diff
@@ -1,28 +1,132 @@
...
-      //               <Chalkboard />
-      //           </Provider>
-      //       </BrowserRouter>);
-        
-
-      // const question = getByText('Question');
-
-      // expect(question).toBeInTheDocument();
-		})
+      const { getByText } = render(<Chalkboard />);
```
Comment (Billwilke42) [inline src/Chalkboard/Chalkboard.test.js:65]: Great job on all these!

Comment (KyleWong2510) [inline src/Chalkboard/Chalkboard.test.js:65]: This is a doozy!  wonder if there is a better way but glad its working

File: src/Chalkboard/Chalkboard.test.js [hunk 3]
```diff
@@ -1,28 +1,132 @@
...
+    });
+
+    it('should not decrease lives when an answer is correct', () => {
+      const { getByText, debug } = render(<Chalkboard />);
+
+      const answer = getByText('Jakarta');
+
+      fireEvent.click(answer);
+      
+    });
```
Comment (KyleWong2510) [inline src/Chalkboard/Chalkboard.test.js:130]: Great job!

**Pull request merged**